src/ir/context.rs

```diff
@@ -1070,6 +1071,7 @@ impl<'ctx> BindgenContext<'ctx> {
                 let name = item.canonical_path(self)[1..].join("::");
                 debug!("whitelisted_items: testing {:?}", name);
                 match *item.kind() {
+                    ItemKind::ObjCInterface(_) => true, // TODO:
```

Comment:
Any particular reason you have a full `ItemKind` for this?

Does an interface have the same characteristics as, let's say, a `struct` in C? I mean, can you forward-declare them, point to them, etc?

If so, you probably want to just fold it into `Type` instead, since you'd share a lot of code for detecting duplicates, going to the definition of the type, etc.

(Edit: after reviewing the rest of the commit I've seen you've also added `TypeKind::ObjCInterface`, you might want to just use `ItemKind::Type` if there isn't a better reason to have a full `ItemKind`).